v2.2/follower-reads.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Follower Reads
+summary: To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature. The trade-off here is that the response might not have the most recent version of the data, though the data will be consistent at the point in time the read is served from.
+
+{{site.data.alerts.callout_danger}}
+The Follower Read feature is only available to [enterprise](https://www.cockroachlabs.com/product/cockroachdb/) users.
+{{site.data.alerts.end}}
+
+## Settings
+
+### Enable/disable Follower Reads
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_split.by_load_enabled` to:
+
+- `false` to disable follower reads _(default)_
+- `true` to enable follower reads 
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.closed_timestamp.follower_reads_enabled = true;
+~~~
+
+#### When to use Follower Reads
+
+Follower Reads return consistent historical reads; currently a minimum of 48 seconds in the past, though we are actively working on reducing that number.
+
+As long as your `SELECT` operations can tolerate slightly outdated data, Follower Reads can reduce read latencies.
+
+#### When not to use Follower Reads
+
+Long-running write operations can have negative interactions with Follower Reads because the Follower Read can either get blocked by the long-running write or cause it to be retried. However, this can be mitigated by placing Follower Reads in `HIGH` priority transactions and using [transaction retries](transactions.html#client-side-transaction-retries) for the write operation.
```

Comment:
This is more of a gotcha than a time not to use follower reads. The time not to use follower reads is when you need fresh data.

Reply:
I feel like your below section is good enough for this issue.